src/main/java/frc/robot/commands/states/ScoringAlgae.java

```diff
@@ -29,13 +32,15 @@ public ScoringAlgae(StateMachine subStateMachine, AlgaeIntake subAlgaeIntake, LE
   @Override
   public void initialize() {
     globalStateMachine.setRobotState(StateMachine.RobotState.SCORING_ALGAE);
-    subAlgaeIntake.setAlgaeIntakeMotor(constAlgaeIntake.ALGAE_OUTTAKE_SPEED);
     globalLED.setLED(constLED.LED_SCORING_ALGAE);
   }
 
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
+    if (globalElevator.isAtSetPoint() && subAlgaeIntake.isAtSetpoint() && subAlgaeIntake.isAtSetRotation()) {
+      subAlgaeIntake.setAlgaeIntakeMotor(constAlgaeIntake.ALGAE_OUTTAKE_SPEED);
+    }
```

Comment:
Is there a reason to have subAlgaeIntake.isAtSetpoint() and subAlgaeIntake.isAtSetRotation() they look like the same code:
![image](https://github.com/user-attachments/assets/dda53482-1da1-499e-b350-783361968b20)